public/assets/js/scripts.js

```diff
@@ -267,22 +299,251 @@ $(document).ready(function () {
+    periodicChanged = true;
+    enableSeachCalculateBtn(periodicChanged, initialChanged);
+  });
+  $(".initial-investment-input").on("input", function () {
+    initialChanged = true;
+    enableSeachCalculateBtn(periodicChanged, initialChanged);
   });
 
+  $(".search-calculate-button").click(function () {
+    const periodic = $(".periodic-investment").val();
+    const initial = $(".initial-investment").val();
+    const avgPropertyCost = removeCommasAndMakeNumber(
+      $(".investment-value-input").data("max-amount")
+    );
+    const frequency = $("input[name='investment-frequency']:checked").val();
+
+    const balance = avgPropertyCost - initial;
+    let output = [];
+    let recommendation = "";
+    let recommendationNav = "";
+    let recommendationBody = "";
+
+    let popOverIcon = `<a tabindex="0" class="" id="search-ready-awesome-spread" role="button" data-toggle="popover">
+                        <img src="./assets/img/icons/question-mark.svg" alt="payment">
+                      </a>`;
+
+    // Recommendations breakdown can be found here - https://docs.google.com/document/d/1gsomOY9qclUz9RzadN3ztJH4Y0ryGT-fukNBFLhJAIU/edit?pli=1#heading=h.yy9lcow7gkem
+
+    const outrightPaymentPersonal =
+      initial >= avgPropertyCost * 0.5 && balance / periodic < 6 / frequency;
+    const completeSpreadPayment =
+      initial >= avgPropertyCost * 0.2 &&
+      balance / periodic > 6 / frequency &&
+      balance / periodic <= 24 / frequency;
+    const outrightMortgagePMI =
+      initial >= avgPropertyCost * 0.25 &&
+      avgPropertyCost + 2500000 - initial <= 45000000;
+    const outrightMortgageNHF =
+      initial >= avgPropertyCost * 0.1 &&
+      avgPropertyCost + 2500000 - initial <= 15000000;
+    const assistedMortgagePMI =
+      periodic * frequency * 24 >= avgPropertyCost * 0.25 &&
+      avgPropertyCost + 2500000 - initial > 15000000 &&
+      avgPropertyCost + 2500000 - initial <= 45000000;
+    const assistedMortgageNHF =
+      periodic * frequency * 24 >= avgPropertyCost * 0.25 &&
+      avgPropertyCost + 2500000 - initial <= 15000000;
+    const rentToOwn =
+      initial >= avgPropertyCost * 0.05 &&
+      periodic >= avgPropertyCost * 0.01 &&
+      balance / periodic <= 120 / frequency;
+    const assistedRentToOwn =
+      periodic >= avgPropertyCost * 0.01 &&
+      balance / periodic <= 120 / frequency;
+    const hybrid =
+      initial >= avgPropertyCost * 0.01 && balance / periodic <= 24 / frequency;
+
+    const packages = {
+      outrightPayment: {
+        title: `Outright`,
+        advice:
+          "With no extra credit option, you're one step away from owning your home",
+      },
+      spreadPayment: {
+        title: `Spread`,
+        advice: "Pay as low as 20%, and spreading the remaining balance within",
+      },
+      oPmiMortgage: {
+        title: `PMI Mortgage`,
+        advice: "You're eligible for a PMI mortgage",
+      },
+      aPmiMortgage: {
```

Comment:
Can we write this in full so that it is more explanatory and can easily be mapped to the right category in the document? Please correct the other categories below too.
```suggestion
      assistedPmiMortgage: {
```